icu4c/source/i18n/unitconverter.cpp

```diff
@@ -176,6 +176,11 @@ double strHasDivideSignToDouble(StringPiece strWithDivide, UErrorCode &status) {
     return strToDouble(strWithDivide, status);
 }
 
+} // namespace
+// FIXME/TODO: this namespace closing and reopening hack makes below
```

Comment:
I believe this was me: I need to clean this up, seems I didn't do a follow-up I intended.

Reply:
Okay, no problem

Reply:
Please move the extractCompoundBaseUnit implementation down or up, out of the anonymous namespace, so that it doesn't need to be closed and reopened above and below (i.e. fix the two FIXMEs).

I did this "temporary hack" to minimize the number of changes, and chance of further merge conflicts, when I did a "git merge". (That was pulling in unitsdata.cpp. UnitsRouter needed access to it to be able to call "getUnitCategory".)